src/ecs/src/lib.rs

```diff
@@ -1,9 +1,10 @@
 extern crate time;
 extern crate specs;
 
-mod processor;
 mod sim;
 
-pub use specs::*;
-pub use self::processor::{Processor, ProcessorResult};
+use time::Duration;
+
+pub use specs::{World, Component, System as Processor, RunArg, Entity, EntityBuilder, Entities,
```

Comment:
if you just do `pub use specs;`, the users would neither need to specify a version